connexion/decorators/uri_parsing.py

```diff
@@ -174,6 +175,8 @@ def resolve_form(self, form_data):
                 self._resolve_param_duplicates(form_data[k], encoding, 'form')
             if defn and defn["type"] == "array":
                 form_data[k] = self._split(form_data[k], encoding, 'form')
+            elif encoding.get('contentType', None) == 'application/json':
```

Comment:
```suggestion
            elif 'contentType' in encoding && utils.all_json([encoding.get['contentType'])]:
```

It is not perfect, the point is that you are reusing utility from connexion.
How do you feel about that?